src/app/src/components/EmbeddedMapFieldsConfig.jsx

```diff
@@ -271,10 +331,24 @@ function EmbeddedMapFieldsConfig({
                                             />
                                         }
                                         label="Select All"
+                                        style={{ width: '312px' }}
                                         classes={{
                                             label: classes.listText,
                                         }}
                                     />
+                                    <Tooltip title="Choose which visible fields to include in text search for your map. Facility name, address, and OAR ID will always be included.">
```

Comment:
Consider increasing the font size of this tooltip. It is a bit hard to read at it's current size.